### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by leon-lopes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,38 @@
 # DevOps Culture
 
-TODO
+### WHAT IS
+This project describes the implementation of devops culture in a software development team. The aim of project is to brings students into devops culture by developing an TODO application applying devops techniques such as the develop flow, package, build and deploy flow (Continuous integration and Delivery), software testing documeting the roles involved in the process.
```
Comment (toledompm) [inline README.md]:
```suggestion
This project describes the implementation of devops culture in a software development team. The aim of project is to brings students into devops culture by developing a TODO application applying devops techniques such as the development flow, package, build and deploy flow (Continuous integration and Delivery), software testing documenting the roles involved in the process.
```

**Review by toledompm — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,7 +1,38 @@
...
+- Selection of tools
+- Basic definition of versioning processes
+- Fundamentals of tests and test types
+- Fundamentals of environments and environment types
+
+### Sprint 2
+- Basic application building process
+- Unit tests and Integration testing
+- Cloud Introduction
+- ECS
```
Comment (jasondavindev) [inline README.md]:
```suggestion
- Application running in cloud environment
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,38 @@
...
+
+### Repositories
+[Application](https://github.com/toledompm/devops-app)
+
+### TECNOLOGIES
+- Node.js
+- Jest
+- Docker
+- ECS
+- EC2
```
Comment (jasondavindev) [inline README.md]: For now, is not necessary. We can add to application repository

File: README.md [hunk 4]
```diff
@@ -1,7 +1,38 @@
...
+
+### TODO
```
Comment (jasondavindev) [inline README.md]:
```suggestion
### Development pillars
```

File: README.md [hunk 5]
```diff
@@ -1,7 +1,38 @@
 # DevOps Culture
 
-TODO
+### WHAT IS
+This project describes the implementation of devops culture in a software development team. The aim of project is to brings students into devops culture by developing an TODO application applying devops techniques such as the develop flow, package, build and deploy flow (Continuous integration and Delivery), software testing documeting the roles involved in the process.
 
+### TEAM
+* [Jason Davin Oliveira Carneiro](https://www.linkedin.com/in/jason-carneiro/)
```
Comment (jasondavindev) [inline README.md]:
```suggestion
* [Jason Carneiro](https://www.linkedin.com/in/jason-carneiro/)
```
:bike:

**Review by M-Fernandes — APPROVED** [review]

**Pull request merged**